scipy/stats/_continuous_distns.py

```diff
@@ -6269,6 +6269,17 @@ def _ppf(self, q, skew):
         ans[invmask] = sc.gammaincinv(alpha, q[invmask])/beta + zeta
         return ans
 
+    @extend_notes_in_docstring(rv_continuous, notes="""\
+        Note that method of moments (`method='MM'`) is not
```

Comment:
probably want to make these lower case too.

Reply:
The intent was to keep these capitalized. If there is consensus that method identifiers be lowercase, though, I'll change it.

Reply:
>If there is consensus that method identifiers be lowercase, though, I'll change it.

I don't think so, but I could be wrong. Fine to leave as is for now, easy to change later.